drivers/at/at.c

```diff
@@ -102,11 +154,16 @@ int at_recv_bytes_until_string(at_dev_t *dev, const char *string,
         }
         else {
             *bytes_len = len;
-            return res;
+            break;
```

Comment:
If you break here, the line above is not need since there's the same thing when exiting the while loop.

https://github.com/RIOT-OS/RIOT/pull/14327/files#diff-61e7246e5a3ec44797fee220e411c8d0R160

Reply:
Removed